README.md

```diff
@@ -4,14 +4,15 @@ Basic starter kit to create your own provisioning
 
 ## Quick start
 
-1. Clone this repo `git clone https://github.com/personal-provisioning/starterkit.git`
-2. Delete the git folder `rm -rf .git/`
-3. Init git again `git init`
-4. Run `setup.sh`
+1. Setup the directory strucutre: `mkdir -p ~/.personal_provisioning`
+2. Clone this repo `git clone https://github.com/personal-provisioning/starterkit.git ~/.personal_provisioning/config`
```

Comment:
You don't really have to specify this. You can clone it anywhere.

Reply:
You are right, will change it 👍